test/test_runner/test_runner.cpp

```diff
@@ -67,11 +68,18 @@ void TestRunner::testStatement(TestStatement* statement, Connection& conn,
     const auto actualResult = conn.query(statement->query);
     QueryResult* currentQueryResult = actualResult.get();
     idx_t resultIdx = 0u;
-    do {
-        checkLogicalPlan(conn, currentQueryResult, statement, resultIdx);
-        currentQueryResult = currentQueryResult->getNextQueryResult();
-        resultIdx++;
-    } while (currentQueryResult);
+    if (TestHelper::REWRITE_TESTS)
+        do {
+            writeOutput(currentQueryResult, statement, resultIdx);
```

Comment:
Hmm, quite a bit of code duplication. How about flipping the `if` and `do` and only perform the `if` for line 73/79?